src/test/java/de/mediathekview/mserver/base/utils/UrlUtilsTestGetUrlParameterValue.java

```diff
@@ -0,0 +1,42 @@
+package de.mediathekview.mserver.base.utils;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Optional;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class UrlUtilsTestGetUrlParameterValue {
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][] { 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)